### PR TITLE
Added deploy script

### PR DESCRIPTION
**Pull request by KmaTTi89**
Need that deploy script in repository to test with TC

### CODE REVIEW

File: tools/deploy.ps1 [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Param ($checkoutdir, $user, $password, $organization, $server, $projectName)
+
+$StarCounterDir = "$checkoutdir\sc"
```
Comment (warpech) [inline tools/deploy.ps1]: We never use such capitalisation `StarCounter`. Always `Starcounter`.

File: tools/deploy.ps1 [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+try 
+{
+	if (Test-Path $StarpackExePath)
+	{
+		if (-Not (Test-Path Env:\StarcounterBin))
+		{ 
+			echo "StarcounterBin variable not found, creating..."
+			[Environment]::SetEnvironmentVariable("StarcounterBin", "$StarCounterDir", "Process") 
+		}
+		else { echo "StarcounterBin variable found, proceeding..." }
```
Comment (warpech) [inline tools/deploy.ps1]: Does starpack need StarcounterBin env var?

File: tools/deploy.ps1 [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+		$createPack = Start-Process -FilePath $StarpackExePath -ArgumentList "-p" -PassThru -Wait -NoNewWindow
+		if ($createPack.ExitCode -eq 0)
+		{
+			echo "File created"			
+			$file = Get-ChildItem -Path "*.zip" -Include "*$projectName*"
+			if (!$file) { exit(1) }
+			$uploadPack = Start-Process -FilePath $StarpackExePath -ArgumentList "-u $file -s=$server -o=$organization -u=$user -p=$password" -PassThru -NoNewWindow -Wait
+			if ($uploadPack.ExitCode -eq 0)
+			{
+				echo "File sended"
```
Comment (warpech) [inline tools/deploy.ps1]: "File sent"

File: tools/deploy.ps1 [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+				exit(0)
+			}
+			else { exit(1) }
+		}
+		else { exit(1) }					
+	}
+	else 
+	{ 
+		echo "Starpack.exe no found, exiting..."
```
Comment (warpech) [inline tools/deploy.ps1]: "not found"

File: tools/deploy.ps1 (hunk 2 above)
Comment (KmaTTi89) [inline tools/deploy.ps1]: This is why I wrote this part of code

File: tools/deploy.ps1 (hunk 1 above)
Comment (KmaTTi89) [inline tools/deploy.ps1]: Fixed

File: tools/deploy.ps1 (hunk 3 above)
Comment (KmaTTi89) [inline tools/deploy.ps1]: Fixed

File: tools/deploy.ps1 (hunk 4 above)
Comment (KmaTTi89) [inline tools/deploy.ps1]: Fixed

**Pull request merged**